variables.tf

```diff
@@ -1,84 +1,72 @@
 variable "helm_config" {
-  type        = any
   description = "Helm provider config for the ondat addon"
```

Comment:
Consistent ordering:
- Description of the variable
- The data type/structure
- The default value if any